auth-web/src/views/auth/AccountDeactivate.vue

```diff
@@ -253,14 +253,20 @@ export default class AccountDeactivate extends Vue {
       this.$refs.successModal.open()
     } catch (err) {
       // eslint-disable-next-line no-console
-      console.error(err)
-      switch (err?.response?.status) {
-        case 400:
-          this.message = err.response.data.message
-          break
-        default:
-          this.message =
-            'An error occurred while attempting to deactivate your account.Please try again'
+      if (err?.response?.status === 400) {
+        switch (err.response.data?.code) {
+          case 'OUTSTANDING_CREDIT':
```

Comment:
not a huge improvement , but can use Map to keep this mapping and do a get using err.response.data?.code

Reply:
done